File: Primer año/Programacion I/1 - Algoritmo de la vida cotidiana/js/compras.js
const faltantesComida = [
    "jamon",
    "queso",
    "pascualina",
]
const faltantesOtros = [
    "bujía", 
    "tornillos"
]
    
const mediosDeTransporte = () =>{
    const solitude = prompt("Cuál será tu medio de transporte? 1. Auto 2. Caminando 3. Bici")
    if(solitude == 1){
        alert("Acordate de las llaves! Y revisa el agua")
    }else if(solitude == 2 ){
        alert("Bien, te toca hacer ejercicio asi que ponete ropa comoda")
    }else if(solitude == 3){
        alert("La cleta, buen ejercicio. Deberás llevar una mochila o algo para cargar los elementos de manera comoda. Las bolsas pueden meterse entre las ruedas.")
    }
    
    const trazadoDeViaje = () =>{
        const viaje = prompt("Introduce el viaje que realizarás")
    }

    
} 

const establecimientos = () =>{
    if(faltantesComida.length > 0 && faltantesOtros.length > 0){
        alert(`Debes ir a diversos establecimientos porque tienes los siguientes elementos: 
        1: ${faltantesComida}. 2:${faltantesOtros}`)
    }
    mediosDeTransporte()
}

establecimientos()

File: Primer año/Programacion I/1 - Algoritmo de la vida cotidiana/py/compras.py
# faltantesComida = [
#     "jamon",
#     "queso",
#     "pascualina",
# ]
    
# faltantesOtros = [
#     "bujía", 
#     "tornillos"
# ]

def definirFaltantes(param): 
    print("Revisa que hace falta en la casa")
    
    print(f"Hacen falta {param}? 1. Si 2.No?")
    confirmComida = int(input()) 
    if confirmComida == 1: 
        print("Escribe los alimentos que deberás conseguir")
        faltantesComida = input()
    
    print("Hace falta algun otro artículo del hogar? 1.Si 2.No")
    confirmOtros = int(input()) 
    if confirmOtros == 1:
        print("Escribe los artículos que deberás conseguir")
        faltantesOtros = input() 
        
    print(f"Deberas conseguir lo siguiente: 1. Comida: {faltantesComida} 2. Otros: {faltantesOtros}")
    return faltantesComida, faltantesOtros
    
definirFaltantes("alimentos")
definirFaltantes("artículo del hogar")

def mediosDeTransporte(): 
    print("Cual sera tu medio de transporte? 1. Auto 2.Caminando 3.Bici ")
    solitude = int(input())
    
    if solitude == 1 : 
        print("Acordate de las llaves! Y revisa el agua")
    elif solitude == 2: 
        print("Bien, te toca hacer ejercicio asi que ponete ropa comoda")
    elif solitude == 3:
        print("La cleta, buen ejercicio. Deberás llevar una mochila o algo para cargar los elementos de manera comoda. Las bolsas pueden meterse entre las ruedas.")
    else: 
        print("Disculpa, ese medio de transporte no esta disponible")

def establecimientos(): 
    if len(faltantesComida) >= 0 & len(faltantesOtros) >= 0:
        print(f"Debes ir a diversos establecimientos porque tienes los siguientes elementos: 1. {faltantesComida} 2.{faltantesOtros}")
        
    mediosDeTransporte()
    
